corehq/apps/userreports/expressions/json_query_specs.py

```diff
@@ -0,0 +1,76 @@
+import itertools
+from dimagi.ext.jsonobject import JsonObject, DictProperty, StringProperty
+from corehq.apps.userreports.exceptions import BadSpecError
+from corehq.apps.userreports.specs import TypeProperty
+from .utils import SUPPORTED_UCR_AGGREGATIONS, aggregate_items
+
+
+class FilterItemsExpressionSpec(JsonObject):
+    type = TypeProperty('filter_items')
+    items_expression = DictProperty(required=True)
```

Comment:
anything that is intended to be an expression should be `DefaultProperty` so that it also accepts literals.